src/c/brokerless/brokerless.c

```diff
@@ -384,13 +418,13 @@ bool listen_brokerless(uxrSession* session, int timeout)
         int32_t hash_index = find_brokerless_hash_from_hash_only_reader(hash);
 
         if (-1 != hash_index && brokerlessEntityMap.queue[hash_index].id.type != UXR_DATAWRITER_ID)
-        {    
+        {
             uxrObjectId * object_id = &brokerlessEntityMap.queue[hash_index].id;
 
             // TODO (pablogs9): request_id is related to the uxr_buffer_request_data request, so it can determine some limitations imposed into the communication -> NOT IMPLEMENTED BY NOW
             if (object_id->type == UXR_DATAREADER_ID)
             {
-                uxrStreamId stream = {0, 0, UXR_BROKERLESS, UXR_INPUT_STREAM};
+                uxrStreamId stream = {0, 0, UXR_BROKERLESS_STREAM, UXR_INPUT_STREAM};
```

Comment:
Check this, have you changed the stream type enum also?

Reply:
yes, I just did it to maintain the coherence with the naming of the rest of the elements in the enum :slightly_smiling_face: